src/components/tiptap/floatingMenu/FloatingMenu.tsx

```diff
@@ -127,8 +155,17 @@ export const FloatingMenu = forwardRef((props: any, ref: any) => {
 
   const { items } = props
 
+  const menuRef = useRef<HTMLDivElement | null>(null)
+
   return (
-    <div className='flex flex-col gap-0.5 bg-white py-2 border border-new-card-border rounded shadow-vairant-1 w-48 overflow-hidden relative'>
+    <div
+      className='flex flex-col gap-0.5 bg-white py-2 border border-new-card-border rounded shadow-vairant-3 w-48 relative'
+      style={{
+        maxHeight: '200px',
```

Comment:
along with `maxHeight` we can calculate available height for the container through some DOM query (i.e. offset of the cursor position to top of the window or bottom of the window) and set height accordingly to avoid overflow

Reply:
current I have used tippy library to do the thing  mentioned in above comment. please find it  [HERE](https://github.com/pagevamp/copilot-client-home/blob/17231c957175cc389742db76cdbe2e3e138945cf/src/components/tiptap/floatingMenu/floatingMenuSuggestion.tsx#L139-L162)

However, if we don't set the `maxHeight` then the menu will overflow because the height of the menu is so large (as demonstrated in the video in PR details). In the design, the menu should not have maxHeight - i.e. menu should appear full. But as demonstrated in the video, having full menu will have overflow at some point. @saugatmaharjan